config/settings/base.py

```diff
@@ -18,7 +18,8 @@
 APPS_DIR = ROOT_DIR.path("metaci")
 
 env = environ.Env()
-env.read_env()
+env_file_path = Path(ROOT_DIR, '.env')
+env.read_env(str(env_file_path))
```

Comment:
@jstvz why is this needed?

Outside of Docker, we tell people to copy env.example to env (no initial period) and source it before running things. That's needed to add our virtualenv and node bin to the path but also takes care of adding variables to the environment.

Inside Docker, docker-compose.yml sets the env_file to .env so it should take care of setting the environment variables.

Reply:
From 6f12711:

    Got the following warning:
    
    ```
    30 0.310 /usr/local/lib/python3.8/site-packages/environ/environ.py:628: UserWarning: /app/config/settings/.env doesn't exist - if you're not configuring your environment separately, create one.
    ```
    
    ...two reasons: 1) The container can't see the environment variables set
    by docker-compose until CMD runs, so it's not available in the build; 2)
    django-environ's logic for finding the project root relies on
    sys._getframe() and is broken (I think).
---
That said, I was pretty iffy about it, so I'm open to an alternate approach.

Reply:
@jstvz It works for me without this when building with BUILD_ENV=dev (so we don't try to run the Django collectstatic command). I see the warning but it's not a problem (we aren't expecting the variables to come from django-environ reading this file).

For BUILD_ENV=production I do get an error due to the missing env vars, but I think we need to do more work on a solution so that we don't end up embedding secrets in the image. Either we need a way to collect static assets without loading Django settings, or we need a way to use secrets while building without keeping them in the image (see https://pythonspeed.com/articles/docker-build-secrets/). I'd like to come back to this later since my main goal in this PR is to fix local builds, and we aren't using the Docker image in production.

Reply:
I'm fine with backing this out then.